app/src/main/java/com/codesoom/assignment/application/TaskControllerAdvice.java

```diff
@@ -0,0 +1,26 @@
+package com.codesoom.assignment.application;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * 할 일들의 예외처리를 Http 상태로 리턴하는 클래스.
```

Comment:
클래스 주석에 `클래스`라고 적을 필요는 없습니다.
```suggestion
 * 할 일과 관련된 예외처리를 담당합니다.
```

Reply:
작성하면서 고민했던 내용인데 굳이 안넣어도 되는군요!